.github/actions/setup-sentry/action.yml

```diff
@@ -0,0 +1,125 @@
+name: 'Sentry Setup'
+description: 'Sets up a Sentry test environment'
+inputs:
+  python:
+    description: 'python version (2 or 3)'
+    required: false
+    default: '3'
+  snuba:
+    description: 'Is snuba required?'
+    required: false
+    default: '1'
+  kafka:
+    description: 'Is kafka required?'
+    required: false
+    default: '0'
+
+outputs:
+  yarn-cache-dir:
+    description: "Path to yarn cache"
+    value: ${{ steps.config.outputs.yarn-cache-dir }}
+  pip-cache-dir:
+    description: "Path to pip cache"
+    value: ${{ steps.pip-cache.outputs.pip-cache }}
+  acceptance-dir:
+    description: "Path to acceptance visual snapshot artifacts"
+    value: ${{ steps.config.outputs.acceptance-dir }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup default environment variables
+      shell: bash
+      env:
+        PYTHON_VERSION: ${{ inputs.python }}
+        NEED_KAFKA: ${{ inputs.kafka }}
+      run: |
+        echo "PIP_DISABLE_PIP_VERSION_CHECK=on" >> $GITHUB_ENV
+        echo "MIGRATIONS_TEST_MIGRATE=0" >> $GITHUB_ENV
+        echo "SENTRY_LIGHT_BUILD=1" >> $GITHUB_ENV
+        echo "SENTRY_SKIP_BACKEND_VALIDATION=1" >> $GITHUB_ENV
```

Comment:
```suggestion
        cat >> $GITHUB_ENV <<EOF
PIP_DISABLE_PIP_VERSION_CHECK=on
MIGRATIONS_TEST_MIGRATE=0
SENTRY_LIGHT_BUILD=1
SENTRY_SKIP_BACKEND_VALIDATION=1
EOF
```

Reply:
And similarly for other places. Anything conditional can be separate echos.

Reply:
IMO what billy has is easier to read

Reply:
Okay, it's not worth the effort then.